lib/mix/tasks/expire_jobs.ex

```diff
@@ -0,0 +1,18 @@
+defmodule Mix.Tasks.ExpireJobs do
+  use Mix.Task
+  alias Companies.{Repo, Schema.Job}
+  import Ecto.Query, warn: false
+
+  @shortdoc "Expires all jobs that haven't been updated for a month."
+  def run(_) do
+    [:postgrex, :ecto]
+    |> Enum.each(&Application.ensure_all_started/1)
```

Comment:
No pipe here.

You could also just make this `Application.ensure_all_started(:companies)` and remove the start_link below.